stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -326,12 +341,11 @@ public void setUsage(@Usage String usage) {
         putStripeJsonModelMapIfNotNull(hashMap, FIELD_RECEIVER, mReceiver);
         putStripeJsonModelMapIfNotNull(hashMap, FIELD_REDIRECT, mRedirect);
 
-        if (mType != null) {
-            hashMap.put(mType, mSourceTypeData);
-        }
+        String usableType = toUsableType(mType, mCustomType);
+        hashMap.put(usableType, mSourceTypeData);
```

Comment:
Here, if we have `dogecoin` as our useable type, we will have the "SourceTypeData" as a map that is found under key "dogecoin", just as the bitcoin params have key "bitcoin", etc.